pkg/utils/url.go

```diff
@@ -0,0 +1,29 @@
+package utils
+
+import (
+	"io/ioutil"
+	"net/http"
+)
+
+// GetFileFromURL retrieves the content of the file on the given url
+func GetFileFromURL(url string) ([]byte, bool, error) {
+	resp, err := http.Get(url)
+	if err != nil {
+		return nil, false, err
+	}
+	if resp.StatusCode < 200 || resp.StatusCode >= 300 {
```

Comment:
What about `4xx` and `5xx`?

Reply:
correct me if I'm wrong - the check says:
* `resp.StatusCode < 200` - anything that is lower than 200

or
* `resp.StatusCode >= 300` - anything that is higher or equal 300, which means also 4xx and 5xx

then return empty `[]byte` and boolean as false - which means that the request wasn's sucessful

Reply:
returning error when not matched 525c07a